content/schema/7.md

```diff
@@ -1,74 +1,31 @@
 +++
-date = "2017-04-27T23:05:53+10:00"
+date = "2017-05-01T10:41:33+10:00"
 next = "/schema/8"
 prev = "/schema/6"
-title = "Deleting Data"
+title = "Exercise : Integrating existing data"
 weight = 7
 endpoint = "/mutate"
 +++
 
-There are three deletion options inside a delete mutation.
+We've added a new schema and loaded some company data, but what about
+integrating our previous friends dataset with this company one.
 
-* `<uid> <edge> <uid>/"value" .`  Delete a single triple
-* `<uid> <edge> * .`  Delete all triples for a given edge
-*  `<uid> * * .` Delete all triples for a given node
+Trying to use the blank nodes from previous mutations won't work.  The blank nodes aren't persisted in the store, so when referring to nodes created in a previous mutation, it's the UID that's needed.  So instead of
 
-{{% notice "note" %}}
-  In order to delete with `S * *` pattern. You need to have a Type related to the node itself. See: [Type System](https://docs.dgraph.io/master/query-language/#type-system)
```

Comment:
We should have a similar note like this in the tutorial.

Reply:
I didn't understand this one either. I think this comment seems to be to the schema/8.md